Detector/DetSensitive/src/SimpleDriftChamber.cpp

```diff
@@ -0,0 +1,64 @@
+#include "DetSensitive/SimpleDriftChamber.h"
+
+// FCCSW
+#include "DetCommon/DetUtils.h"
+
+// DD4hep
+#include "DDG4/Geant4Mapping.h"
+#include "DDG4/Geant4VolumeManager.h"
+
+// CLHEP
+#include "CLHEP/Vector/ThreeVector.h"
+
+// Geant4
+#include "G4SDManager.hh"
+
+#include "DetCommon/Geant4PreDigiTrackHit.h"
+
+namespace det {
+SimpleDriftChamber::SimpleDriftChamber(const std::string& aDetectorName,
+                                       const std::string& aReadoutName,
+                                       const dd4hep::Segmentation& aSeg)
+    : G4VSensitiveDetector(aDetectorName), m_driftChamberCollection(nullptr), m_seg(aSeg) {
+  // name of the collection of hits is determined byt the readout name (from XML)
+  collectionName.insert(aReadoutName);
+}
+
+SimpleDriftChamber::~SimpleDriftChamber() {}
+
+void SimpleDriftChamber::Initialize(G4HCofThisEvent* aHitsCollections) {
+  // create a collection of hits and add it to G4HCofThisEvent
+  // deleted in ~G4Event
+  m_driftChamberCollection =
+      new G4THitsCollection<fcc::Geant4PreDigiTrackHit>(SensitiveDetectorName, collectionName[0]);
+  aHitsCollections->AddHitsCollection(G4SDManager::GetSDMpointer()->GetCollectionID(m_driftChamberCollection),
+                                      m_driftChamberCollection);
+}
+
+bool SimpleDriftChamber::ProcessHits(G4Step* aStep, G4TouchableHistory*) {
+  // check if energy was deposited
+  G4double edep = aStep->GetTotalEnergyDeposit();
+  G4double stepLength = aStep->GetStepLength();
+
+  // cuts on the Edep and the G4 step length 
+  if (edep < 10 * CLHEP::eV || stepLength < 5 * CLHEP::micrometer) {
```

Comment:
Oh, I meant just putting the constants 10 eV and 5 microns in the header https://github.com/HEP-FCC/FCCSW/pull/316/files/aa087303bfe840dc519d703b529debc74cbc0d81#diff-2d21ee718bf09e9d7b0f4a0d3a66abdbR47. It's ok if it's  hardcoded, but it will be easier to find there